server.py

```diff
@@ -50,11 +50,45 @@ def parse_range(range_header, content_length):
         end_byte = int(end)
     return start_byte, end_byte
 
+innertube_client_id = settings.innertube_client_id
+innertube_client = util.innertube_client
+client = innertube_client[innertube_client_id]
 def proxy_site(env, start_response, video=False):
+    client_params = util.INNERTUBE_CLIENTS[client]
+    client_context = client_params['INNERTUBE_CONTEXT']
+    client_ua = client_context.get('userAgent') or util.mobile_user_agent
     send_headers = {
-        'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; Win64; x64)',
+        'User-Agent': client_ua,
         'Accept': '*/*',
+        'Accept-language': 'en-US',
+        'X-YouTube-Client-Name': client_params['INNERTUBE_CONTEXT_CLIENT_NAME'],
+        'X-YouTube-Client-Version': client_context['client']['clientVersion'],
     }
+    visitor_data_file = settings.data_dir + '/visitorData.txt'
+    visitor_data = None
+    if not settings.use_po_token:
+        if settings.use_visitor_data:
```

Comment:
It didn't occur in my mind when I was modifying this to use `and` statement. Thanks for pointing out this better approach.

Reply:
I really mean nested if statement instead of adding `settings.use_visitor_data` as additional condition for loading visitorData file.

There was a wrong indentation at the next `else` statement.